firstvoices/backend/views/search/base_search_views.py

```diff
@@ -206,8 +208,28 @@ def get_search_params(self):
 
         return search_params
 
+    def get_pagination_params(self):
+        """
+        Function to return pagination params
+        """
+        default_page_size = self.paginator.get_page_size(self.request)
+
+        page = int(self.request.GET.get("page", 1))
+        page_size = int(self.request.GET.get("pageSize", default_page_size))
+        start = (page - 1) * page_size
+        end = start + page_size
+
+        pagination_params = {
+            "page_size": page_size,
+            "page": page,
+            "start": start,
+            "end": end,
```

Comment:
minor: pagination_params["end"] can be removed since we are using from and size in the ES query.